app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -1589,10 +1590,13 @@ private void hideContextMenus() {
 
     public void startDownload(@NonNull DownloadJob downloadJob, boolean showConfirmDialog) {
         if (showConfirmDialog) {
+            // As of O, the prefixes are used in their standard meanings in the SI system, so kB = 1000 bytes, MB = 1,000,000 bytes, etc.
+            // In Build.VERSION_CODES.N and earlier, powers of 1024 are used instead, with KB = 1024 bytes, MB = 1,048,576 bytes, etc.
+            String fileSize = Formatter.formatFileSize(getContext(), downloadJob.getContentLength());
             mWidgetManager.getFocusedWindow().showConfirmPrompt(
                     R.drawable.ic_icon_downloads,
                     getResources().getString(R.string.download_confirm_title),
-                    downloadJob.getFilename(),
+                    downloadJob.getFilename() + " (" + fileSize + ")",
```

Comment:
You can use HTML here. For example, adding `<br><br>` between the file name and the size looks like this:

![image](https://github.com/Igalia/wolvic/assets/1412060/2041d0ae-a65a-4797-b645-31495feb0c96)

Another thing that we should fix is to allow the title to have more than one line of text. The fix for that is https://github.com/Igalia/wolvic/pull/776

Reply:
Finally, we should probably not show anything if the size is zero.